routes/index.js

```diff
@@ -2,8 +2,8 @@ var express = require('express');
 var router = express.Router();
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+router.get('/dashboard', function(req, res, next) {
```

Comment:
Yeah I know. What do you think a good solution would be? redirect to the dashboard on a authenticated request for "/"?